setup.cfg

```diff
@@ -0,0 +1,29 @@
+[flake8]
+max-line-length = 88
+max-complexity = 18
+select = B,C,E,F,W,T4,B9,B950
```

Comment:
is this file even used for anything else than bandit? in the settings.json are also different settings

Reply:
this is used by all linters, now settings and setup.cfg are the same

Reply:
but it is not used by all linters... in the settings it is just set for bandit...

Reply:
it is used for all linters by pre commit